source/features/highest-rated-comment.tsx

```diff
@@ -33,7 +33,18 @@ function init(): false | void {
 	const avatar = select('.timeline-comment-avatar', event)!.cloneNode(true);
 	const {hash} = select<HTMLAnchorElement>('.timestamp', event)!;
 
-	select('.unminimized-comment', event)!.classList.add('rgh-highest-rated-comment');
+	const comment = select('.unminimized-comment', event)!;
+
+	comment.classList.add('rgh-highest-rated-comment');
+
+	select('.timeline-comment-header .timeline-comment-header-text', event)!.parentElement!.lastElementChild!.before((
```

Comment:
Is there a more direct way to add this label?

Reply:
```ts
select('.timeline-comment-header-text', event)!.before((
```

Reply:
Also, the reason I put before the `-header-text` is that it's parent sorts in reversed order meaning the header itself being the last child will be the leftmost item.